pkg/ebpf/c/tracee.bpf.c

```diff
@@ -4728,6 +4729,25 @@ int BPF_KPROBE(trace_security_inode_mknod)
     return events_perf_submit(&data, SECURITY_INODE_MKNOD, 0);
 }
 
+SEC("kprobe/register_chrdev_region")
+int BPF_KPROBE(trace_register_chrdev_region)
+{
+    event_data_t data = {};
+    if (!init_event_data(&data, ctx))
+        return 0;
+
+    if (!should_trace(&data.context))
+        return 0;
+
+    u32 major_number = PT_REGS_PARM1(ctx);
```

Comment:
Seems like the second parameter is not the major number but the device count:
https://elixir.bootlin.com/linux/v5.17.3/source/fs/char_dev.c#L200

Reply:
Yes u right, i changed to first trace `__register_chrdev` and fixed the prams.
BTW , I saw that the real major number is the return of the function, do you have any idea how we can get the return value of the event?